source/shadow.c

```diff
@@ -154,13 +154,13 @@ static ShadowStatus_t validateMatchTopicParameters( const char * pTopic,
  * @return Return SHADOW_SUCCESS if the parameters are valid;
  *         return SHADOW_BAD_PARAMETER if not.
  */
-ShadowStatus_t validateAssembleTopicParameters( ShadowTopicStringType_t topicType,
-                                                const char * pThingName,
-                                                uint8_t thingNameLength,
-                                                const char * pShadowName,
-                                                uint8_t shadowNameLength,
-                                                char * pTopicBuffer,
-                                                uint16_t * pOutLength );
+static ShadowStatus_t validateAssembleTopicParameters( ShadowTopicStringType_t topicType,
+                                                       const char * pThingName,
+                                                       uint8_t thingNameLength,
+                                                       const char * pShadowName,
+                                                       uint8_t shadowNameLength,
+                                                       const char * pTopicBuffer,
+                                                       const uint16_t * pOutLength );
```

Comment:
It's a `const` output parameter? If this function is only validating inputs then these should be `@param[in]`

Reply:
Yea. good catch. Thanks